human.txt

```diff
@@ -31,8 +32,8 @@ jsstrn
 justyn-clark
 kenhrmt
 kylecho
-Leo-Mu
 leo424y
+Leo-Mu
```

Comment:
Can you fix it please? Once you commit those changes we'll be able to merge your PR. Thanks @danedavid